crates/re_log_types/src/path/entity_path_filter.rs

```diff
@@ -259,6 +259,69 @@ impl EntityPathFilter {
 
         false // no matching rule and we are exclude-by-default.
     }
+
+    /// Checks whether this results of this filter "fully contain" the results of another filter.
+    ///
+    /// If this returns `true` there should not exist any [`EntityPath`] for which [`Self::is_included`]
+    /// would return `true` and the other filter would return `false` for this filter.
+    ///
+    /// This is a conservative estimate, and may return `false` in situations where the
+    /// query does in fact cover the other query. However, it should never return `true`
+    /// in a case where the other query would not be fully covered.
```

Comment:
an example of when that happens would be useful

Reply:
if there's only certain entities present this might be wrong / overly conservative. I figure that's what you meant